jupyter-js-widgets/src/manager-base.ts

```diff
@@ -310,9 +314,22 @@ abstract class ManagerBase<T> {
                     return widget_model;
                 });
         }, function(error) {
-            delete that._models[model_id];
-            var wrapped_error = new utils.WrappedError('Could not create model', error);
-            return Promise.reject(wrapped_error);
+            let modelOptions = {
+                widget_manager: that,
+                model_id: model_id,
+                comm: options.comm,
+            }
+            var widget_model = new HTMLModel({}, modelOptions);
+            widget_model.once('comm:close', function () {
+                delete that._models[model_id];
+            });
+            let placeholder =
+                `<div>
+                  <div>Could not create model:</div>
+                  <div> + ${error.message} +  </div>
```

Comment:
Also, can we add model_name, model_module, and model_module_version in the error message to help in debugging?